src/libstd/net/ip.rs

```diff
@@ -120,7 +120,14 @@ impl Ipv4Addr {
         self.octets()[0] == 169 && self.octets()[1] == 254
     }
 
+    /// Returns true if this address refers to source hosts on "this"
+    /// network. This property defined by RFC 5735.
+    pub fn is_this_network(&self) -> bool {
+        self.octets()[0] == 0
+    }
+
     /// Returns true if the address appears to be globally routable.
+    /// http://www.iana.org/assignments/iana-ipv4-special-registry/iana-ipv4-special-registry.xhtml
```

Comment:
You probably want to make it a proper link? Like 

> See [iana-ipv4-special-registry](http://www.iana.org/assignments/iana-ipv4-special-registry/iana-ipv4-special-registry.xhtml).

with

```
See [iana-ipv4-special-registry][ipv4-sr].

[ipv4-sr]: http://www.iana.org/assignments/iana-ipv4-special-registry/iana-ipv4-special-registry.xhtml
```

or something?